### DISCUSSION TITLE
Rebuild website when push new commit to Pull Request

### DISCUSSION THREAD

**Discussion by YevheniiRybak** [tighten/laravel-deploy-preview#14] [Ideas]
Could you please add a feature for start rebuild when push new commit to Pull Request?

**Comment by bakerkretzmar** [comment]
Preview sites have quick deploy enabled, so they will already automatically rebuild when you push new commits.

**Reply by YevheniiRybak** [reply]
Yeah, indeed. However, I don't see the status of a rebuild in Github.

**Reply by bakerkretzmar** [reply]
Hmmm, yeah I guess that's true. What would you expect here? A new comment on the PR saying the rebuild finished? Or maybe we could update the existing comment to show the last time it was rebuilt or something...

**Reply by YevheniiRybak** [reply]
I suppose it could be as new GitHub action with progress and then update the existing comment.

For example, this feature worked on Vercel or Netlify.
